### DISCUSSION TITLE
custom color of underlined input but without affecting the validation part with danger (Help me, please)

### DISCUSSION THREAD

**Discussion by ManuelCarrascal** [heroui-inc/heroui#3006] [Help]
I have been trying to customize the inputWrapper of my input with an underlined variant for hours and I have not found a way to make it have a specific color but without affecting, for example, the color flows such as danger, I try to enter the discord to seek help and it is blocked Does anyone know how?

![image](https://github.com/nextui-org/nextui/assets/54221416/bafb11b4-d379-4f9e-97da-8f86e0e4f98f)
![image](https://github.com/nextui-org/nextui/assets/54221416/e2988b35-5089-4ef1-aef1-239327443a0e)
but 
![image](https://github.com/nextui-org/nextui/assets/54221416/8e8bed11-7dec-4a73-a5a6-ec80c2d84d0b)
In that case it should be red, and I've tried many things and nothing works for me.

I want the color orange but if there is an error or validation in the format or whatever it looks normal like this:
![image](https://github.com/nextui-org/nextui/assets/54221416/82c1dc46-1518-46bd-896b-121dbe8b0cc4)

**Comment by wingkwong** [comment]
for the error message, you can customize in `errorMessage`. For the label, I saw the styles got `important` property in order to override group data styles. Therefore, you need to add that to label and input too.

```ts
isInvalid: {
  true: {
    label: "!text-danger",
    input: "!placeholder:text-danger !text-danger",
  },
}
```

**Reply by ManuelCarrascal** [reply]
I was referring to the border of the input, but I think I did it by writing the class although it doesn't seem comfortable to me since it is an after element

**Reply by ManuelCarrascal** [reply]
It would be a plus to allow customizing styles of each part of the component in a simpler way without overwriting or damaging the flow.

**Reply by wingkwong** [reply]
Sorry. i totally misread the problem statement. Lemme convert this to an issue.

**Reply by ManuelCarrascal** [reply]
Ok, focus more on the solution part so you can modify pseudo element like that later more easily.
